plugins/linear-genome-view/src/LinearGenomeView/index.tsx

```diff
@@ -566,11 +566,11 @@ export function stateModelFactory(pluginManager: PluginManager) {
           trackId,
         )
         if (!configuration) {
-          throw new Error(`Could not resolve identifier`)
+          throw new Error(`Could not resolve identifier ${trackId}`)
```

Comment:
Suggestion to wrap identifier in quotes

```suggestion
          throw new Error(`Could not resolve identifier "${trackId}"`)
```